### PR TITLE
install.py: warn if the repo isn't up to date

### PR DESCRIPTION
**Pull request by multun**

### CODE REVIEW

File: install.py [hunk 1]
```diff
@@ -957,8 +957,50 @@ def sync_users():
             system(cmd)
 
 
+def git_check():
+    if not sys.stdout.isatty():
+        return True
+    try:
+        git_process = subprocess.Popen(
```
Comment (Zopieux) [inline install.py]: You don't need to communicate with stdin, so just use `subprocess.check_output` and remove all that `PIPE` crap.

File: install.py [hunk 2]
```diff
@@ -957,8 +957,50 @@ def sync_users():
...
+    except:
+        pass
+    return True
+
+
 if __name__ == '__main__':
     os.umask(0)  # Trust our chmods.
+
+    if not git_check():
+        sys.exit(0)
```
Comment (Zopieux) [inline install.py]: Don't return a successful error code when something failed.

Comment (multun) [inline install.py]: I thought of it as a meant behavior (it's user interuption after all).
I'll change it, thanks !

**Comment by multun** [comment]
nonsense

**Pull request closed without merging**